autoload/airline.vim

```diff
@@ -214,7 +214,7 @@ function! airline#check_mode(winnr)
       let l:mode = ['replace']
     elseif l:m[0] =~# '\v(v|V||s|S|)'
       let l:mode = ['visual']
-    elseif l:m ==# "t"
+    elseif l:m ==# "t" && has("termninal")
```

Comment:
Typo. Besides, I don't think this check is actually needed. `l:m` can only ever be `t`, if your vim supports the terminal.

Reply:
I see.